treebeard-lib/treebeard/notebooks/commands.py

```diff
@@ -166,17 +166,18 @@ def zip_filter(info: tarfile.TarInfo):
 
     size = os.path.getsize(src_archive.name)
     max_upload_size = "100MB"
-    if size > parse_size(max_upload_size):
-        fatal_exit(
-            click.style(
-                (
-                    f"ERROR: Compressed notebook directory is {format_size(size)},"
-                    f" max upload size is {max_upload_size}. \nPlease ensure you ignore any virtualenv subdirectory"
-                    " using `treebeard run --ignore venv`"
-                ),
-                fg="red",
+    if not local:
```

Comment:
let's just do this after the local run statement

Reply:
moved after local run statement